packages/paste-core/components/tabs/src/Tab.tsx

```diff
@@ -6,39 +6,71 @@ import {TabPrimitive} from '@twilio-paste/tabs-primitive';
 import {TabsContext} from './TabsContext';
 
 type Orientation = 'horizontal' | 'vertical' | undefined;
+type Variant = 'flexed' | undefined;
```

Comment:
Thinking about this more and how it ties into the orientation. 

For a while I was thinking that orientation should be variants as they are significantly different visual appearances. But then I was wondering how that plays with `flexed`, do vertical tabs get flexed? I don't think they actually play well together.

I'm sure flexed only applies to horizontal.

I'm wondering if vertical tabs should be their own component, and not a variant. 🤔

Reply:
@SiTaggart as we discussed, I've added the TODO note about vertical tabs, and removed the vertical tabs from the doc page.